os/arch/arm/src/amebad/amebad_serial.c

```diff
@@ -374,6 +374,25 @@ static u32 uart_index_get(PinName tx)
 	return 3;
 }
 
+static void LOGUART_PutChar_RAM(u8 c)
+{
+	UART_TypeDef* LOGUART = UART2_DEV;
+	u32 CounterIndex = 0;
+
+	if (ConfigDebugClose == 1)
+		return;
+
+	while(1) {
+		CounterIndex++;
+		if (CounterIndex >=6540)
```

Comment:
Coding rule violation*
>= 6540

Reply:
@thapav I am sorry but we don't want to apply our coding rules for arch, board and external folders.